### DISCUSSION TITLE
European GitHub VAT registration number?

### DISCUSSION THREAD

**Discussion by marcoieni** [community/community#102804] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi 👋
I need to submit an invoice for my monthly sponsors earnings. I live in Italy.
What's GitHub European VAT registration number?
[Here](https://gist.github.com/mitsuhiko/a2c385c5f3b73a07bb35) I found it's EU528001391, but the link is old, so I wanted to confirm. 🙏

**Comment by AndreaGriffiths11** [comment, marked as answer]
Hi there, our VAT registration number is EU528001391. That is a general EU VAT number. We are registered as a single point of registration (SOR) in the Netherlands and pay VAT-collecting jurisdictions via their portal. I hope this helps!

**Reply by marcoieni** [reply]
Thanks Andrea 🙏

**Reply by georgjaehnig** [reply]
Hi Andrea, I tried to validate your VAT number EU528001391 on https://ec.europa.eu/taxation_customs/vies/#/vat-validation (I picked NL as country, as there was no option EU) and got a negative result.

Do you also have a country-based VAT ID? Because in my tax form (Czechia), I need to pick a specific EU country. There is no general "EU" option.

**Reply by jozefizso** [reply]
I would expect GitHub and Microsoft had sort this accounting and taxation already:

<img width="1133" alt="Screenshot 2025-03-19 at 22 18 59" src="https://github.com/user-attachments/assets/74cf7d14-05ca-44a5-ad94-d99e320cc87c" />

**Reply by litio2001** [reply]
No, I confirm that they have not resolved it yet.😒

**Reply by rubensousa** [reply]
Looks like the correct VAT number is: 854265259B01
From: https://www.creditsafe.com/business-index/en-ie/company/github-bv-nl03765938

![image](https://github.com/user-attachments/assets/573f61b2-38d0-4310-9130-de9760cddc90)

**Reply by litio2001** [reply]
thanks!

**Reply by SleeplessByte** [reply]
You cannot validate the EU number through the VIES VAT number validation and as far as I know, there is no guidance on how to validate it. This goes for *any* VAT number that starts with `EU`, which is used by non-eu companies that do EU-taxes.

**Comment by Luk-z** [comment]
How the hell github emit an invoice without their vat number in the header??? lost half day for that, this is so ridicolous and 
disgusting!

This is the header in the invoice....
```
GitHub, Inc.
88 Colin P. Kelly Jr. Street
San Francisco, CA 94107
United States

https://support.github.com/contact

* VAT/GST paid directly by GitHub, where applicable
```

**Reply by georgjaehnig** [reply]
Yeah, so that header shows that they are US based. Thus my current understanding is they don't have an actual EU VAT ID, thus payments from them are not under EU reverse charge, but payments from outside EU.

**Reply by Luk-z** [reply]
It's clear it's not an EU company.
When you register payments from outside EU, is the seller VAT number not required? 
I think it is required.
I need US VAT number to register my invoice, why they do not show the VAT number? is it a secret?

**Reply by iurgi-pixel** [reply]
Hi all, I'm facing same issue. Could you get invoice with vat number?

**Reply by lukzed** [reply]
use a fake vat number, it seems no one care about it so i won't hurt myself anymore.

**Comment by W3bL4b** [comment]
We have need to have the bill with indication of VAT id or VIES or something else

**Reply by havemsho-dev** [reply]
Please transfer the amount to Egypt
